lib/DxilPIXPasses/DxilShaderAccessTracking.cpp

```diff
@@ -795,87 +795,6 @@ DxilShaderAccessTracking::GetResourceFromHandle(Value *resHandle,
   return ret;
 }
 
-static bool CheckForDynamicIndexing(OP *HlslOP, LLVMContext &Ctx,
-                                    DxilModule &DM) {
```

Comment:
Maybe this shouldn't be deleted in this PR cuz it's unrelated to the goals of the PR?